api-reference/api-reference-guide.md

```diff
@@ -0,0 +1,243 @@
+
+Provide a concrete example of the data entity with valid values. Fill in as many optional attributes as possible.
+
+Use fenced code blocks to make your code distinctive from other text blocks.
+
+If you are documenting in Markdown, many Markdown processors also support syntax highlighting, which adds color to keywords. Indicate the language mode of your example to take advantage of this feature. The displayed color schema depends on your processor and the rendering configurations.
+
+### About the "Endpoints" section
+
+List the endpoints that can interact with this resource type in a tabular format with the following guidelines:
+
+- Capitalize the method names, for example "GET".
+- Adopt a consistent naming convention for the APIs. In most cases, the naming convention used in the documentation should be consistent with the one in the source code.
+- Add a link to each of the endpoint names that directs users to the corresponding endpoint reference.
+- For endpoints that are deprecated but still in use, add a note in the “Description” column. Consider using the strikethrough format to indicate the deprecation status.
+
+## About the "API endpoint reference" section
+
+The API endpoint reference section provides reference information for a specific API endpoint. It describes the specifications of an API endpoint, such as the method, URL, request, and response schema.
+
+**Title**
+
+Substitute the document title `{Endpoint Name}` with the actual name.
+
+:information_source:**Tips**:
+
+- Typically the name of an API endpoint consists of the operation type and the resource type. For example, an API endpoint that creates a User resource can be named `Create user`.
+- As in the API resource reference, the naming convention should be consistent throughout your API documentation.
+- Use the singular form of the resource name unless the endpoint is designed exclusively for a bulk operation, such as `List users`.
+
+**Short description**
+
+Provide a one-line description of what the API does. Starts with a verb. For example:
+
+- For "get" operations: `Retrieves a {resource}.`
+- For "list" operations: `Lists {resources}.`
+- For "create" operations: `Creates a {resource}.` or `Inserts a {resource}.`
+- For "update" operations: `Updates a {resource}.`
+- For "delete" operations: `Removes a {resource}.` 
+
+Ensure that the description here is consistent with that listed in the resource reference.
+
+### About the "Endpoint" section
+
+
+Use this section to define the API endpoint.
+
+:information_source: **Tips**:
+
+- Use fenced code blocks to make your code distinctive from other text blocks.
+- Replace {METHOD} with the actual request method and capitalize all letters. For example, `POST`.
+- In the {request_url} segment, start with a slash character `/` and only provide the URL path (the segment after the hostname). The base URL can be omitted if you already document it in the API overview. For example, `/v2/users`.
```

Comment:
**suggestion:** "The base URL can be omitted if you have already documented it..."